v3/newrelic/expect_implementation.go

```diff
@@ -11,9 +11,13 @@ import (
 	"github.com/newrelic/go-agent/v3/internal"
 )
 
-func validateStringField(v internal.Validator, fieldName, v1, v2 string) {
-	if v1 != v2 {
-		v.Error(fieldName, v1, v2)
+func validateStringField(v internal.Validator, fieldName, expect, actual string) {
```

Comment:
Note that there's another `validateStringField` in internal/expect.go. It follows the old "v1 != v2" format.

Reply:
oh, I should fix that too